pkg/server/datastore/sqlstore/models.go

```diff
@@ -96,7 +96,7 @@ type RegisteredEntry struct {
 
 	// Hint is a "hint string" passed to the workload to distinguish between
 	// multiple SVIDs
-	Hint string
+	Hint string `gorm:"index"`
```

Comment:
Yikes. We'll need to add a migration for this. Since it won't impact correctness, we're probably still ok shipping this feature for 1.6.0, but it doesn't mean searching by hint will be slow until they apply migrations.